elements/rh-footer/rh-footer-lightdom.css

```diff
@@ -1,40 +1,42 @@
-rh-footer :is([slot^=links]) a {
-  gap: var(--rh-footer-links-gap, 10px);
+/* ensure links fully wrap img tags */
+:is(rh-footer, rh-global-footer) a[slot^=logo] {
+  display: block;
+}
+
+rh-footer [slot^=links] a {
+  gap: var(--rh-footer-links-gap, 0.625em);
 }
 
-rh-footer :is([slot^=links], [slot=footer-links-primary], [slot=footer-links-secondary]) li {
+:is(rh-footer, rh-global-footer) [slot^=links] li {
   margin: 0;
   padding: 0;
   display: contents;
 }
 
-rh-footer :is([slot^=links], [slot=footer-links-primary], [slot=footer-links-secondary]) a {
-  --_color: var(--link-color, #fff);
-  --_font-size: var(--link-font-size, 14px);
+:is(rh-footer, rh-global-footer) [slot^=links] a {
   display: block;
-  color: var(--_color) !important;
-  font-size: var(--_font-size);
-  text-decoration: none;
+  color: var(--link-color, #fff) !important;
+  font-size: var(--link-font-size, 0.875em);
 }
 
-rh-footer :is([slot^=links], [slot=footer-links-primary], [slot=footer-links-secondary]) a:is(:hover, :focus-within) {
-  text-decoration: underline;
+rh-global-footer [slot^=links] a {
+  font-size: inherit;
```

Comment:
We are telling these links that they should inherit there font-size from inside the web component